tsconfig.json

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
@codingholt any reason the tsconfig was removed?

Reply:
Um no, I don't think so. I must've done it by accident.

Reply:
thanks for pointing it out!